infra/Makefile.toml

```diff
@@ -18,7 +18,7 @@ default_to_workspace = false
 # General network-specific parameters
 SUBNET_ID = { value = "/r0", condition = { env_not_set = ["SUBNET_ID"] } }
 # The network name is derived from the SUBNET_ID, replacing slashes with dashes, and dropping the first dash if any.
-NETWORK_NAME = { script = ["echo $SUBNET_ID | sed -e 's/\\//-/g' -e 's/^-//1'"] }
+NETWORK_NAME = { script = ["echo $SUBNET_ID | sed -e 's|/|-|g' -e 's|^-||1'"] }
```

Comment:
What does it do to change `s/.../.../g` into `s|...|...|g`? 

(I have no idea what the whole expression does, but that's just me being lame).

Reply:
I added this to use a different separator than in the subnet names ("/").

Reply:
The comment already says "replacing slashes with dashes". What are you replacing it with now? Or did it not work? 

I was also asking in more general: what does `|` do compared to `/` with `sed`?

Reply:
`|` or `/` are just separators in sed script. The semantics is the same